operators/pkg/instance-controller/container_logic.go

```diff
@@ -146,6 +147,26 @@ func buildContainerInstanceDeploymentSpec(
 			// LivenessProbe:   &tigerVncProbe,
 			ReadinessProbe: &tigerVncProbe,
 		},
+		{
+			Name:            "filebrowser",
+			Image:           o.FileMgrImg + ":v2.15.0",
```

Comment:
The tag should not be hard-coded but obtained as a parameter

Reply:
Could the tag be a parameter of the enclosing function, or is it better to define it as a field of the "ContainerEnvOpts" struct?

Reply:
I believe it should a field of the ContainerEnvOpts struct, as it needs to be configured with a command-line parameter through the helm manifest in order to allow for fast upgrades without having to modify the source code